outgoing_email_by_model/tests/test_mailserver_by_model.py

```diff
@@ -3,7 +3,8 @@
 
 from odoo_test_helper import FakeModelLoader
 
-from odoo.tests.common import Form, TransactionCase
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
 
 
 class TestMailserverByModel(TransactionCase):
```

Comment:
```suggestion
class TestMailserverByModel(BaseCommon):
```
if relevant: 

> Use `BaseCommon` as base test class to reduce overhead from tracking and boost the test suite.


https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0